src/components/interfaces/MOBILE_API.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>

```

Comment:
@AKalinich-Luxoft 
There are a lot of changes due to formatting which hides real changes so I think that needs to be fixed